stable/prometheus/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: prometheus
-version: 1.3.1
+version: 1.4.0
```

Comment:
@mgoodness @viglesiasce had to update this to fix the formatting when specifying a storage class. What should the version bump be?